Source/WebKit/Shared/LogStream.h

```diff
@@ -60,6 +62,7 @@ class LogStream : public IPC::StreamMessageReceiver {
     RefPtr<IPC::StreamServerConnection> m_logStreamConnection;
     int32_t m_pid { 0 };
     Markable<LogStreamIdentifier> m_logStreamIdentifier;
+    static unsigned m_logCount;
```

Comment:
This seems unnecessary to be data member of `LogStream` if it's only used in `LogStream`; we may move definition of `logCount()` and this to `LogStream.cpp`?

Reply:
The WKWebView testing SPI is calling `logCount()`, so I think we need to keep the `logCount()` method.

Reply:
We can keep `logCount()` declaration here but move its definition to LogStream.cpp, then we don't need `m_logCount ` here. i.e. `unsigned  LogStream::logCount() { return globalLogCount; }` in LogStream.cpp